docs/04-documentation/LZ-04-05.adoc

```diff
@@ -1,17 +1,17 @@
 // tag::DE[]
 [[LG-04-05]]
 ==== LZ 04-05 [ehemaliges LZ 3-04]: Architektursichten erläutern und anwenden (R1)
-Softwarearchitekt:innen können folgende Architektursichten anwenden (siehe <<kruchten95>>, <<rozanskiwoods>>, <<starkelorz>>, <<arc42>>, <<brownc4>>):
+Softwarearchitekt:innen können folgende Architektursichten anwenden:
 
 * Kontextsicht (auch genannt Kontextabgrenzung)
-  ** z.{nbsp}B. in Form von Kontextdiagrammen mit Erläuterungen darstellen
-  ** externe Schnittstellen von Systemen in der Kontextabgrenzung darstellen
-  ** fachlichen und technischen Kontext differenzieren
+  ** enthält die externen Schnittstellen von Systemen
+  ** bei Bedarf differenziert nach fachlichem und technischem Kontext
 
```

Comment:
Die Leerzeile kann raus?

Reply:
I consider the empty line a reading aid for adoc authors.